File: server/src/main/java/books/Book.java
package books;

import java.sql.Date;

public class Book {
	private String id;
	private String title;
	private String publicationDate;
	private String publisher;
	private double price;
	
	public Book(String id, String title, String publicationDate, String publisher, double price) {
		super();
		this.id = id;
		this.title = title;
		this.publicationDate = publicationDate;
		this.publisher = publisher;
		this.price = price;
	}
	
	public String getId() {
		return id;
	}
	public void setId(String id) {
		this.id = id;
	}
	public String getTitle() {
		return title;
	}
	public void setTitle(String title) {
		this.title = title;
	}
	public String getPublicationDate() {
		return publicationDate;
	}
	public void setPublicationDate(String publicationDate) {
		this.publicationDate = publicationDate;
	}
	public String getPublisher() {
		return publisher;
	}
	public void setPublisher(String publisher) {
		this.publisher = publisher;
	}
	public double getPrice() {
		return price;
	}
	public void setPrice(double price) {
		this.price = price;
	}
}

File: client/src/model/book.ts
export class Book {
    private id: string;
    private title: string;
    private author: string;
    private publicationDate;
    private publisher;
    private price;

    constructor(id: string, title: string, author: string, publicationDate, publisher, price) {
        this.id = id;
        this.title = title;
        this.author = author;
        this.publicationDate = publicationDate;
        this.publisher = publisher;
        this.price = price;
    }
}
